src/settings/BatchGroupConfigurationSettings/BatchGroupConfigurationSettings.js

```diff
@@ -6,14 +6,67 @@ import React, {
 import PropTypes from 'prop-types';
 
 import { LoadingPane } from '@folio/stripes/components';
+import {
+  useShowCallout,
+} from '@folio/stripes-acq-components';
 
-import { batchGroupsResource } from '../../common/resources';
+import { EXPORT_CONFIGURATIONS_API } from '../../common/constants';
+import {
+  batchGroupsResource,
+  credentialsResource,
+  exportConfigsResource,
+} from '../../common/resources';
+import { SCHEDULE_EXPORT } from './constants';
+import { saveExportConfig } from './utils';
 import BatchGroupConfigurationForm from './BatchGroupCongiurationForm';
 
 const BatchGroupConfigurationSettings = ({ mutator }) => {
   const [isLoading, setIsLoading] = useState(true);
   const [batchGroups, setBatchGroups] = useState();
   const [selectedBatchGroupId, setSelectedBatchGroupId] = useState();
+  const [exportConfig, setExportConfig] = useState();
+  const [credentials, setCredentials] = useState();
+  const showCallout = useShowCallout();
+
+  const fetchExportConfig = useCallback(
```

Comment:
actually, it's a wrong usage of `useCallback`. You can treat it as `useMemo` and that not what you want here, IMHO. And it looks like this should be under `useEffect`

Reply:
I need to call this function after PUT/POST